csharp/ql/src/Security Features/CWE-937/VulnerablePackageGOOD.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Copy this file (and `VulnerablePackageBAD.csproj`) to the test.

Reply:
Currently, qltest doesn't extract `.csproj` files as XML so it wouldn't do anything. I'm not really sure it's worth making that change.

Reply:
OK.

Reply:
OK.

Reply:
OK.